app/autotranslate/server/models/Settings.js

```diff
@@ -0,0 +1,21 @@
+import { Settings } from '../../../models/server/models/Settings';
+
+Object.assign(Settings, {
+	renameSetting(oldId, newId) {
```

Comment:
We added this in order to migrate the API-key-setting. The migration itself is pending though - right, @vickyokrm ?